### PR TITLE
CDAP-7151. Fixed map reduce classpath for the explore container by ad…

### PR DESCRIPTION
**Pull request by sagarkapare**
…ding cdap-common.jar in the beginning.

Build is running here - http://builds.cask.co/browse/CDAP-RUT147-1

### CODE REVIEW

**Comment by sagarkapare** [comment]
@poornachandra Please take a look thanks!

**Comment by sagarkapare** [comment]
verified on cluster.

File: cdap-explore/src/main/java/co/cask/cdap/explore/service/ExploreServiceUtils.java [hunk 1]
```diff
@@ -507,10 +507,15 @@ private static File updateMapredConfFile(File confFile, File tempDir) {
     String mrAppClassPath = conf.get(MRJobConfig.MAPREDUCE_APPLICATION_CLASSPATH,
                                      MRJobConfig.DEFAULT_MAPREDUCE_APPLICATION_CLASSPATH);
 
-    // Add the pwd/* at the beginning of classpath. Without this change, old jars from mr framework classpath
-    // get into classpath.
-    mrAppClassPath = "$PWD/*," + mrAppClassPath;
+    // We first add cdap-common.jar to the classpath so that FileContextLocationFactory from cdap-common gets
```
Comment (bdmogal) [inline cdap-explore/src/main/java/co/cask/cdap/explore/service/ExploreServiceUtils.java:511]: we won't need this when `FileContextLocationFactory` is fixed in Twill, right? If so, create a jira for removing this, and add a TODO?

**Comment by bdmogal** [comment]
one comment about adding a comment. Rest LGTM

File: cdap-explore/src/main/java/co/cask/cdap/explore/service/ExploreServiceUtils.java (hunk 1 above)
Comment (sagarkapare) [inline cdap-explore/src/main/java/co/cask/cdap/explore/service/ExploreServiceUtils.java:511]: added todo.

**Comment by bdmogal** [comment]
lgtm

**Pull request merged**